test/unit.test.js

```diff
@@ -17,8 +17,51 @@ for (const { testName, isRoot } of unitTests) {
     console.log(`Skipping unit test on Windows: ${testSuffix}`);
     continue;
   };
+  const unitPath = join(__dirname, 'unit', testName);
+  
+  if (testName === 'wildcard') {
+    it('should work with async fs options used', async () => {
```

Comment:
How about updating line 98 instead of a whole new test? Something like:

```diff
- readFile: readFileMock,
+ readFile: testName === 'someAsyncTest' ? readFileMockAsync : readFileMock,
```

Reply:
Updated in https://github.com/vercel/nft/pull/236/commits/e04da34f55a34693ae01dba2dfda54a18f0cf56c